### ISSUE TITLE
Double activated Vita?

### ISSUE THREAD

**Issue by sammargh** [ZombieWizzard/vita-activator#6]
I'm not entirely sure how this happened but I used your application (v0.2) last night to activate a Vita. It worked fine and showed up on my account and was great. Today I was grabbing all of my data from PSN through my PS3 and used USB to transfer to the Vita. I first disabled all wifi, plugged into QCMA with offline mode enabled to get it to be recognized on the PS3 via USB.

When I pluged the console into the PS3 an error message came up that I already had 3 Vitas on my account (Old Vita, PSTV, new Vita activated last night and some new one.)

After this I tried using your application to deactivate and every thing came back as Invalid IDPS (you should be really scared). Sufficiently scared I started clicking around and did a hail-mary disabling all devices via account management, re-activating my PS4, PS3 and then this troublesome Vita. This time it activated fine and there's no ghost Vita hanging around.

Again I'm not entirely sure how this happened but all I know is it re-activated when plugged into the PS3 as it didn't think the system was activated enough which then flagged the IDPS until I forced everything deactivated.

I was able to re-activate the other Vita & Playstation TV fine through my PS3 as well so something isn't set right in this application. While it did work I'd advise people to not plug their activated systems into a PS3 until you are able to figure out what happened.

**Comment by sammargh** [comment]
Sorry had the wrong version at first. Also on 2FA if you have never activated the vita the username is not populated so it will not authenticate as it tries to pull it from somewhere and only asks for a device password.

**Comment by ZombieWizzard** [comment]
I'm not sure why it would complain about an invalid idps (it may be that that version had some issue I was unaware of) as a vita should never have that issue.  
About the 2FA thing it's just a matter of using the 'Enter PSN Account info manually' option and inputting your code as the password. I suppose I could add the option to the input 2FA code and user but it would be purely a visual change and the process would be no different. When I get free time I probably will do that.  
Also I'll try and look into the issue with a PS3 you described but as I have no PS3 myself it might be some time before I can properly investigate. For now I'll just flag this as enhancement.  
Thanks for raising the issue

**Comment by sammargh** [comment]
The problem with 2FA was I selected it but it only asked for a device password and login was null with password as device setup key as the system had never been activated. With 2FA on Vita & PS3 you use your regular PSN login as the user and then the device password generated from idp.sn/device-password

**Comment by ZombieWizzard** [comment]
Yeah I'm aware of that, I only added the option for 2FA code input to make it simpler for it to be easier to understand. As I said "it's just a matter of using the 'Enter PSN Account info manually' option" until I get around to changing it to be more obvious.
